docs/maps.md

```diff
@@ -0,0 +1,142 @@
+# Maps
+
+The `Maps` class provides access to the Mapbox Maps API.  You can import it from either the `mapbox` module or the `mapbox.services.maps` module.
+
+__mapbox__:
+
+```python
+>>> from mapbox import Maps
+```
+
+__mapbox.services.maps__:
+
+```python
+>>> from mapbox.services.maps import Maps
+```
+
+See https://www.mapbox.com/api-documentation/#maps for general documentation of the API.
+
+Use of the Maps API requires an access token, which you should set in your environment.  For more information, see the [access tokens](access_tokens.md) documentation.
+
+## Maps Methods
+
+The public methods of the `Maps` class provide access to the Maps API and return an instance of [`requests.Response`](http://docs.python-requests.org/en/latest/api/#requests.Response).
+
+## Usage: Retrieving Tiles
+
+Instantiate `Maps`.
+
+```python
+>>> maps = Maps()
+```
+
+Call the `get_tile` method, passing in values for `map_id`, `z` (zoom), `x` (column), and `y` (row).  Pass in values for optional arguments as necessary - `retina` (double scale), `file_format`, `style_id`, and `timestamp`.
+
+```python
+>>> response = maps.get_tile("mapbox.streets", z=0, x=0, y=0)
+```
+
+Evaluate whether the request succeeded, and retrieve the tile from the response object.
+
+```python
+>>> if response.status_code == 200:
+...     with open("./0.png", "wb") as output:
+...         output.write(response.content)
+```
+
+## Usage: Retrieving HTML Slippy Maps
+
+Instantiate `Maps`.
+
+```python
+>>> maps = Maps()
+```
+
+Call the `get_html_slippy_map` method, passing in a value for `map_id`.  Pass in values for optional arguments as necessary - `options` (map controls and behaviors), `z`, `lat`, and `lon`.
+
+```python
+>>> response = maps.get_html_slippy_map("mapbox.streets")
```

Comment:
I'm not certain about the usefulness of getting a slippy map in Python. I think we could eliminate this method and have less API to support.